astropy/coordinates/builtin_frames/ecliptic_transforms.py

```diff
@@ -37,15 +37,24 @@ def _mean_ecliptic_rotation_matrix(equinox):
 
 
 def _true_ecliptic_rotation_matrix(equinox):
-    # This code calls pnm06a from ERFA, which retrieves the precession
-    # matrix (including frame bias) according to the IAU 2006 model, and
-    # including the nutation. This family of systems is less popular
+    # This code calls the same routines as done in pnm06a from ERFA, which
+    # retrieves the precession matrix (including frame bias) according to
+    # the IAU 2006 model, and including the nutation.
+    # This family of systems is less popular
     # (see https://github.com/astropy/astropy/pull/6508).
     jd1, jd2 = get_jd12(equinox, 'tt')
-    rnpb = erfa.pnm06a(jd1, jd2)
-    _, nut_obl = erfa.nut06a(jd1, jd2)*u.radian
-    obl = erfa.obl06(jd1, jd2)*u.radian + nut_obl  # calculate the true obliquity of the ecliptic
-    return matrix_product(rotation_matrix(obl, 'x'), rnpb)
+    # Here, we call the three routines from erfa.pnm06a separately,
+    # so that we can keep the nutation for calculating the true obliquity
+    # (which is a fairly expensive operation); see gh-11000.
+    # pnm06a: Fukushima-Williams angles for frame bias and precession.
+    gamb, phib, psib, epsa = erfa.pfw06(jd1, jd2)
+    # pnm06a: Nutation components.
+    dpsi, deps = erfa.nut06a(jd1, jd2)
```

Comment:
Similarly here, these are nutation components in longitude and obliquity right? So are `nut_lon` and `nut_obl` better names?

Or if we wanted to keep the consistency with the F-W angle names we could use `nut_psi` and `nut_eps`

Reply:
Here, I agree more with you, but with once one decides on the one above, this one follows (and is clearer, since `dpsi` matches `psi` and `deps` matches `eps` -- this would be true even if going for the F-W names). So, again I thought it easiest to just expand the comment above to `pnm06a: Nutation components (in longitude and obliquity).`